### ISSUE TITLE
JavaScript error for non-required, non-existent array object members

### ISSUE THREAD

**Issue by phispi** [plantain-00/schema-based-json-editor#31]
Thanks for the wonderful project! I ran into an issue with the following JSON schema:

```json
{
  "type": "object",
  "properties": {
    "name": {
      "type": "string",
    },
    "aliases": {
      "type": "array",
      "items": {
        "type": "string",
      }
    }
  }
}
```

The important thing is the `aliases` property and the fact that it is not required. Everything works fine as long as the `aliases` entry is present in the initial value:
```json
value: {
  "name": "Michael",
  "aliases": ["Mike"]
}
```

However, as soon as it is omitted (which is allowed by the schema)
```json
value: {
  "name": "Michael"
}
```

the browser console reports the error `TypeError: _ctx.value is undefined` (Firefox) or `variables.js:75 Uncaught (in promise) TypeError: Cannot read property 'length' of undefined` (Chromium)

The same error happens if the checkbox labeled "not exists" at the array property is checked.

#### Environment: 
* Firefox 78.8.0esr (64-bit); 
* Chromium: 88.0.4324.182 (Developer Build) built on Debian 10.8, running on Debian 10.8 (64-bit)

I hope this is the right place to report this ;-)

**Comment by plantain-00** [comment]
v8.1.1 should fix this

**Comment by phispi** [comment]
Yes, it indeed fixes it, thank you very, very much! Very impressing how fast you investigated and fixed it! :+1:
